### PR TITLE
Add support for create_option = ImportSecure for managed_disk

### PR DESCRIPTION
**Pull request by balazs92117**
"Security Type 'ConfidentialVM_DiskEncryptedWithCustomerKey' is not supported for CreateOption 'Import'. Supported create options are FromImage, ImportSecure, UploadPreparedSecure."

### CODE REVIEW

File: internal/services/compute/managed_disk_resource.go [hunk 1]
```diff
@@ -490,8 +491,9 @@ func resourceManagedDiskCreate(d *pluginsdk.ResourceData, meta interface{}) erro
 		switch createOption {
 		case disks.DiskCreateOptionFromImage:
 		case disks.DiskCreateOptionImport:
+		case disks.DiskCreateOptionImportSecure:
 		default:
-			return fmt.Errorf("trusted_launch_enabled cannot be set to true with create_option %q. Supported Create Options when Trusted Launch is enabled are FromImage, Import", createOption)
+			return fmt.Errorf("trusted_launch_enabled cannot be set to true with create_option %q. Supported Create Options when Trusted Launch is enabled are FromImage, Import, ImportSecure", createOption)
```
Comment (jackofallops) [inline internal/services/compute/managed_disk_resource.go:496]:
```suggestion
			return fmt.Errorf("trusted_launch_enabled cannot be set to true with create_option %q. Supported Create Options when Trusted Launch is enabled are `FromImage`, `Import`, `ImportSecure`", createOption)
```

File: website/docs/r/managed_disk.html.markdown [hunk 2]
```diff
@@ -130,13 +131,13 @@ The following arguments are supported:
 
 ~> **NOTE:** Setting logical sector size is supported only with `UltraSSD_LRS` disks and `PremiumV2_LRS` disks.
 
-* `os_type` - (Optional) Specify a value when the source of an `Import` or `Copy` operation targets a source that contains an operating system. Valid values are `Linux` or `Windows`.
+* `os_type` - (Optional) Specify a value when the source of an `Import`, `ImportSecure` or `Copy` operation targets a source that contains an operating system. Valid values are `Linux` or `Windows`.
```
Comment (jackofallops) [inline website/docs/r/managed_disk.html.markdown:134]: Do we need a note here to let users know `ImportSecure` is valid for ConfidentialVM only?

File: website/docs/r/managed_disk.html.markdown [hunk 3]
```diff
@@ -150,7 +151,11 @@ The following arguments are supported:
 
 -> **Note:** Trusted Launch can only be enabled when `create_option` is `FromImage` or `Import`.
```
Comment (jackofallops) [inline website/docs/r/managed_disk.html.markdown:152]: This note needs updating for `ImportSecure` also?

**Review by jackofallops — CHANGES_REQUESTED** [review]
Hi @balazs92117
Thanks for this PR. I've left some comments in-line below.  Could you also look at adding tests for the new value to verify/ensure this works as expected?

Thanks!

**Review by katbyte — APPROVED** [review]
LGTM ☔

**Pull request merged**